testsuite/openshift/mockserver.py

```diff
@@ -0,0 +1,38 @@
+"""Module for Mockserver integration"""
+import json
+from urllib.parse import urljoin
+
+import requests
+
+
+class Mockserver:
```

Comment:
I would move this somewhere else than openshift dir, since it doesn't have anything to do with openshift

Reply:
Do you have something in mind?

Reply:
If this will be a single file, I would simply put it in `testsuite/` dir